### DISCUSSION TITLE
What about release version?

### DISCUSSION THREAD

**Discussion by SangjiLEE** [eclipse-edc/Connector#1689] [Q&A] [answered]
Hello, 
I would like to ask about the release version.
It seems that 「0.0.1-milestone-4」 is the latest release version.
However, I found the sentence on the README that planning to have actual release versions starting sometime in mid-2022.
maybe, forget to erase the sentence after the release??

and one more, does the release version mean the stable version of EDC which are not like the snapshot version?
![sentence](https://user-images.githubusercontent.com/80669462/178924429-89846197-c747-4183-83d2-3c8d46001ce1.png)

Thank you in advance.

**Comment by ndr-brt** [comment, marked as answer]
That sentence was about the fact that we were about to start releasing artifacts, now we're doing it, probably the README needs to be updated:
https://search.maven.org/search?q=g:org.eclipse.dataspaceconnector
